Changed Dashboard to portrait format, rearranged widgets and added he… (#745)
DataDog/integrations-extras: jfrog_platform/README.md

* Kubernetes Cluster
* JFrog Artifactory and/or JFrog Xray installed via [JFrog Helm Charts][1]
* [Helm 3][2]

> * Datadog's API key (you can get it from Datadog account -> Settings -> API)

Reviewer: ```suggestion
* Your [Datadog API key][4].
```

> ### Step 1 Fluentd Configuration

Reviewer: ```suggestion
### Step 1 Fluentd configuration
```

Datadog's Fluentd plugin can be used to forward logs directly from Fluentd to your Datadog account.

> Fluentd integration is done by specifying the API key as follows:

Reviewer: ```suggestion
Set up the Fluentd integration by specifying the API key as follows:
```

> _api_key_ is the API key from Datadog

Reviewer: ```suggestion
_api_key_ is your [Datadog API key][4].
```

</match>
```

> ### Step 2 Integration Enablement

Reviewer: ```suggestion
### Step 2 Integration enablement
```

> To enable this integration, run the td-agent on `artifactory` pods

Reviewer: ```suggestion
To enable this integration, run the td-agent on `artifactory` pods:
```

Add all attributes as facets from **Facets** > **Add** (on the left side of the screen in Logs) > **Search**.

> ### Step 3 Install JFrog Platform Tile 

Reviewer: ```suggestion
### Step 3 JFrog platform tile 
```

> If you have not installed the JFrog Platform tile yet, install the tile.

Reviewer: ```suggestion
If you have not installed the JFrog platform tile yet, install the tile.
```

> ### Step 4 Open JFrog Artifactory Dashboard

Reviewer: ```suggestion
### Step 4 JFrog Artifactory Dashboard
```

Go to Dashboard -> Dashboard List, find `JFrog Artifactory Dashboard` and open it.

### Step 5 Search for logs

> Logs can be access in the UI under  **Logs** > **Search**. Select the specific source mentioned in the Fluentd configuration (`fluentd` in configuration example above) to retrieve logs.

Reviewer: ```suggestion
Access your [logs][5] in Datadog. Select the specific source mentioned in the Fluentd configuration (`fluentd` in configuration example above) to retrieve logs.
```

[1]: https://github.com/jfrog/charts
[2]: https://helm.sh/

> [3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/jfrog_platform/images/dashboard.png

Reviewer: ```
[4]: https://app.datadoghq.com/account/settings#api
[5]: https://app.datadoghq.com/logs
```

Author: Hello there. I accepted all your changes. Only these two footer additions did not show up with "accept suggestion" button.

Reviewer: @vinayagg - You will need to add them using your text editor

Author: @ruthnaebeck Thank you, I added these references.
